src/content/docs/browser/new-relic-browser/performance-quality/browser-monitoring-performance-impact.mdx

```diff
@@ -252,6 +252,31 @@ The agent consists of multiple scripts to reduce the initial size of the "loader
       </td>
     </tr>
 
+    <tr>
+      <td>
+        Session replay
+      </td>
+
+      <td>
+        no
+      </td>
+
+      <td>
+        yes
+      </td>
+
+      <td>
+        yes
+      </td>
+
+      <td>
+        Session replays are harvested when one of the following occurs:
+        * When the compressed data collected for the session reaches a maximum size of 64 KB.
+        * If the user remains on the same page for more than 60 seconds.
+        * The visibility of the page changes (tab change, focus, navigation, etc)
```

Comment:
The first two bullets complete the sentence started in the pre-colon clause, but the last bullet doesn't.

How about "When the visibility..." instead?